src/nomad_measurements/xrd/schema.py

```diff
@@ -80,19 +80,17 @@
     )
     import pint
 
-from nomad.datamodel.metainfo.eln.nexus_data_converter import populate_nexus_subsection
-from pynxtools import dataconverter
 
 m_package = Package(name='nomad_xrd')
 
 
 
 def calculate_two_theta_or_q(
-    wavelength: 'pint.Quantity',
-    q: 'pint.Quantity' = None,
-    two_theta: 'pint.Quantity' = None,
-) -> tuple['pint.Quantity', 'pint.Quantity']:
-    """
+        wavelength: 'pint.Quantity',
+        q: 'pint.Quantity'=None,
+        two_theta: 'pint.Quantity'=None
+    ) -> tuple['pint.Quantity', 'pint.Quantity']:
+    '''
```

Comment:
Are you using any code formater like `ruff` or `black` if yes what is the which one and what is the version of that. It is cumbersome doing manually as I always use `ruff` format of code on save which reduces effort in code maintaining. Because in the other PR you will see same issue again.

Reply:
To avoid running the ruff formatting, you can set this setting in `.vscode/setting.json`:
```json
"editor.formatOnSave": false
```

Reply:
yeah, totally, I also rely on it heavily. But at least for this PR, it's better to turn it off.